example_responses/supplier_response.json

```diff
@@ -21,14 +20,24 @@
     "description": "Supplier Description",
     "dunsNumber": "1234",
     "eSourcingId": "1234",
+    "companiesHouseNumber": "1234578",
+    "registrationCompany": "country:GB",
+    "tradingStatus": "limited company (LTD)",
+    "organisationSize": "medium",
+    "VAT Number": "GB 123 4567 89",
```

Comment:
`VAT Number`? That's long gone 🤔

Reply:
Hmm, that's funny, it was in a response I was looking at... I guess the supplier was created a while ago. Shall I remove?